### PR TITLE
Added link to app on GH pages

### PR DESCRIPTION
**Pull request by sverhoeven**

### CODE REVIEW

File: run-your-c++-code-on-the-web/README.md [hunk 1]
```diff
@@ -248,6 +248,8 @@ From the figure at the top of the article, the root of the equation should be at
 
 _The resulting page if everything works._
 
+If you can not or do not want to host the app yourself then visit a version hosted on [GitHub pages](https://nlesc-jcer.github.io/run-cpp-on-web/run-your-c++-code-on-the-web/index.html).
```
Comment (jspaaks) [inline run-your-c++-code-on-the-web/README.md]:
```suggestion
_The resulting page if everything works. See the live version of the app on [GitHub pages](https://nlesc-jcer.github.io/run-cpp-on-web/run-your-c++-code-on-the-web/index.html)._
```

**Review by jspaaks — CHANGES_REQUESTED** [review]
Made a suggestion for a change

**Comment by jspaaks** [comment]
Good to merge.

File: run-your-c++-code-on-the-web/README.md (hunk 1 above)
Comment (sverhoeven) [reply run-your-c++-code-on-the-web/README.md]: Nice suggestion, making it part of caption for #34 .1

**Review by jspaaks — APPROVED** [review]

**Pull request merged**